docs/developer-docs/latest/setup-deployment-guides/configurations/required/middlewares.md

```diff
@@ -111,12 +101,12 @@ Strapi's core includes the following internal middlewares, mostly used for perfo
 - responses, which handle the [responses](/developer-docs/latest/development/backend-customization.html#responses),
 - [security](#security),
 - [public](#public),
-- [session](#session)
+<!-- - [session](#session) -->
 
 <!-- TODO: remove this comment — the link to backend custom > responses above won't work until merged with  PR #446 -->
 
 ::: caution
-The following middlewares cannot be disabled: `responses`, `router`, `logger` and `error`.
+The following middlewares cannot be disabled: `errors`, `security`, `cors`, `query`', `body`, `public`, `favicon`.
```

Comment:
I think this deserves calrification. From v4 on they are put in the configuration and should not be removed but can (that will throw and error saying a middleware is missing) it's a bit different than in v3.

Reply:
Thanks! [Clarified it](https://github.com/strapi/documentation/pull/455/commits/d1ee7e34b6715481f22ddf195297e22534bf8795)